docs/designers-developers/developers/block-api/block-edit-save.md

```diff
@@ -406,4 +406,4 @@ When a block is detected as invalid, a warning will be logged into your browser'
 
 **I've changed my block's `save` behavior and old content now includes invalid blocks. How can I fix this?**
 
-Refer to the guide on [Deprecated Blocks](/docs/designers-developers/developers/block-api/block-deprecations.md) to learn more about how to accommodate legacy content in intentional markup changes.
+Refer to the guide on [Deprecated Blocks](https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-deprecation/) to learn more about how to accommodate legacy content in intentional markup changes.
```

Comment:
```suggestion
Refer to the guide on [Deprecated Blocks](/docs/designers-developers/developers/block-api/block-deprecation.md) to learn more about how to accommodate legacy content in intentional markup changes.
```

Good catch! We should use the relative link to get this paged also linked on GitHub.

Reply:
cheers for sorting that